denops/skkeleton/deps/std/assert.ts

```diff
@@ -1 +1 @@
-export * from "jsr:@std/assert@~1.0.3";
+export { assertEquals } from "jsr:@std/assert@~1.0.3/equals";
```

Comment:
Are there any differences between other removed files and `deps/std/assert.ts`?

Reply:
OK.  I have removed it.